packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -166,6 +166,68 @@ export class RestAPIClient {
 		return response.json();
 	}
 
+	/**
+	 * Gets the latest of post from the blogs a user follows.
+	 *
+	 *
+	 * @returns {Promise<ReaderParams}.
+	 * @throws {Error} If API responded with an error.
+	 */
+	async getReaderFeed(): Promise< ReaderParams > {
+		const params: RequestParams = {
+			method: 'get',
+			headers: {
+				Authorization: await this.getAuthorizationHeader( 'bearer' ),
+				'Content-Type': this.getContentTypeHeader( 'json' ),
+			},
+		};
+
+		const response = await this.sendRequest( this.getRequestURL( '1.1', '/me/sites' ), params );
```

Comment:
**comment**: the `/me/sites` endpoint return an array of site data, represented by the interface `AllSitesResponse`, not `ReaderParams`.

This may be one of the reasons why the code doesn't work currently.

Reply:
I have no idea how `/me/sites` got in there instead of `read/following` 😅 It's all fixed now though!